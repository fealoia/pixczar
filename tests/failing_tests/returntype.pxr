Void main() {
	Int x = 0;
	return x;
}
