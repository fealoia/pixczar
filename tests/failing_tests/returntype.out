Fatal error: exception Failure("return gives Int expected Void in x")
